app/src/main/java/org/thunderdog/challegram/telegram/TdlibAccount.java

```diff
@@ -495,6 +496,7 @@ public static class DisplayInformation {
       } else {
         this.profilePhotoSmallPath = this.profilePhotoBigPath = null;
       }
+      this.isPremium = user.isPremium;
       saveAll();
```

Comment:
Purpose of `DisplayInformation` is to have information about the account without need in `Tdlib` instance awakening. `saveAll` and `fullRestore` methods must support new fields properly (`isPremium` in this case), otherwise they won't have correct values in most cases.